File: 03 problem set 1/03 mario-more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int num;
    do
    {
        num = get_int("Height: "); // input height
    }
    while (num < 1 || num > 8);

    for (int hash = 1; hash <= num; hash++)
    {
        for (int i = 1; i <= num - hash; i++)
        {
            printf(" ");
        }

        for (int j = 1; j <= hash; j++)
        {
            printf("#");
        }

        printf("  ");

        for (int j = 1; j <= hash; j++)
        {
            printf("#");
        }

        printf("\n");
    }
}

File: 18 problem set 6/03 sentimental-mario-more/mario.py
# TODO
from cs50 import get_int


def main():
    num = get_int("Height: ")
    while num < 1 or num > 8:
        num = get_int("Height: ")
    for hash in range(1, num + 1):
        for i in range(num - hash):
            print(" ", end="")
        for j in range(hash):
            print("#", end="")
        print("  ", end="")
        for k in range(hash):
            print("#", end="")
        print()


main()
